index.js

```diff
@@ -140,4 +144,15 @@ document.getElementById('closebtnDSK').addEventListener('click', () => {
   appearDeskModel.style.display = 'none';
   deskModal.classList.remove('active');
   Deskoverlay.classList.remove('active');
-});
\ No newline at end of file
+});
+
+form.addEventListener('submit', (event) => {
+  let a = false;
+  if (email.value === email.value.toLowerCase()) {
+    a = true;
```

Comment:
- [ ] it would be better if you give `a` variable better name than a so everyone ends up file can understand what this variable used for

Reply:
Thankyou @youmari i have applied the suggestion!